osc/iterm2.go

```diff
@@ -45,10 +83,15 @@ func initCellSize() {
 		return
 	}
 	defer terminal.Restore(1, s)
-	fmt.Fprint(os.Stdout, ecsi+"1337;ReportCellSize"+st)
-	fileSetReadDeadline(os.Stdout, time.Now().Add(time.Second))
-	defer fileSetReadDeadline(os.Stdout, time.Time{})
-	fmt.Fscanf(os.Stdout, "\033]1337;ReportCellSize=%f;%f\033\\", &cellHeight, &cellWidth)
+	if !sixelEnabled {
```

Comment:
Let's isolate that into an `initSixelCellSize` and rename this one `initIterm2CellSize` and call the appropriate one depending on the case.